docs/index.md

```diff
@@ -8,36 +8,73 @@
   </a>
 </p>
 
-**Welcome!** 👋 Explore metadata terms used by MC<sup>2</sup> Center
-project by key, value, or descriptions.
+### **Welcome!**  
+The MC<sup>2</sup> Center Data Models Explorer enables structured, standardized documentation of data and resources for research projects like the [Cancer Complexity Knowledge Portal](https://www.cancercomplexity.synapse.org/). 
```

Comment:
change "for research projects like the [Cancer Complexity Knowledge Portal](https://www.cancercomplexity.synapse.org/)"
to
"for the [Cancer Complexity Knowledge Portal](https://www.cancercomplexity.synapse.org/). "